packages/material-ui/src/Tabs/Tabs.test.js

```diff
@@ -101,158 +90,141 @@ describe('<Tabs />', () => {
-      assert.notStrictEqual(tab0, tab1);
-      const wrapper = mount(
-        <Tabs width="md" onChange={noop} value={tab0}>
+      expect(tab0).to.not.equal(tab1);
+
+      const { getAllByRole } = render(
+        <Tabs value={tab0}>
           <Tab value={tab0} />
           <Tab value={tab1} />
         </Tabs>,
       );
-      const tabWrappers = wrapper.find('[role="tab"]').hostNodes();
-
-      assert.isOk(tabWrappers.at(0).props()['aria-selected']);
-      assert.isNotOk(tabWrappers.at(1).props()['aria-selected']);
-    });
-
-    it('should switch from the original value', () => {
-      const wrapper = mount(tabs);
-      wrapper.setProps({ value: 0 });
-
-      const tabWrappers = wrapper.find('[role="tab"]').hostNodes();
-
-      assert.isOk(tabWrappers.at(0).props()['aria-selected']);
-      assert.isNotOk(tabWrappers.at(1).props()['aria-selected']);
+      const tabElements = getAllByRole('tab');
+      expect(tabElements[0]).to.have.attribute('aria-selected', 'true');
+      expect(tabElements[1]).to.have.attribute('aria-selected', 'false');
     });
 
     describe('indicator', () => {
       it('should accept a false value', () => {
-        const wrapper = mount(
-          <Tabs width="md" onChange={noop} value={false}>
+        const { container } = render(
+          <Tabs value={false}>
             <Tab />
             <Tab />
           </Tabs>,
         );
-        assert.strictEqual(wrapper.find(TabIndicator).props().style.width, 0);
+        expect(container.querySelector(`.${classes.indicator}`).style.width).to.equal('0px');
       });
 
       it('should let the selected <Tab /> render the indicator server-side', () => {
-        const markup = render(
-          <Tabs width="md" onChange={noop} value={1}>
+        const markup = serverRender(
+          <Tabs value={1}>
             <Tab />
             <Tab />
           </Tabs>,
         );
         const indicator = markup.find(`button > .${classes.indicator}`);
-        assert.strictEqual(indicator.length, 1);
+        expect(indicator).to.have.lengthOf(1);
       });
 
       it('should render the indicator', () => {
-        const wrapper = mount(
-          <Tabs width="md" onChange={noop} value={1}>
+        const { container, getAllByRole } = render(
+          <Tabs value={1}>
             <Tab />
             <Tab />
           </Tabs>,
         );
-        assert.strictEqual(
-          wrapper
-            .find(Tab)
-            .at(0)
-            .props().indicator,
-          false,
-        );
-        assert.strictEqual(
-          wrapper
-            .find(Tab)
-            .at(1)
-            .props().indicator,
-          false,
-        );
-        assert.strictEqual(wrapper.find(TabIndicator).length, 1);
+        const tabElements = getAllByRole('tab');
+        expect(tabElements[0].querySelector(`.${classes.indicator}`)).to.not.be.ok;
+        expect(tabElements[0].querySelector(`.${classes.indicator}`)).to.not.be.ok;
```

Comment:
Did you mean to check a different index? Looks like it should've been 1.

Reply:
Yes, thanks